lesson_00/kevinmason/README.md

```diff
@@ -0,0 +1,53 @@
+# Kevin-s_Read_me
```

Comment:
Please change the title here (without dashes and underscored).

Reply:
changed and took dashes out